"Adafruit_VS1053.cpp" problem with "feather_player" exemple

Arduino board: Adafruit ESP32 Feather + Music Maker FeatherWing 

Arduino IDE version : 1.8.9

Hi, There is a message error when I try to upload the "feather_player" sketch. 


Here is the message error : 

`/Users/romain/Documents/Arduino/libraries/Adafruit_VS1053/Adafruit_VS1053.cpp: In member function 'long unsigned int Adafruit_VS1053_FilePlayer::mp3_ID3Jumper(fs::File)':
/Users/romain/Documents/Arduino/libraries/Adafruit_VS1053/Adafruit_VS1053.cpp:186:25: error: invalid conversion from 'char*' to 'uint8_t* {aka unsigned char*}' [-fpermissive]
       if (mp3.read(tag,3)) {

                         ^
In file included from /Users/romain/Documents/Arduino/hardware/espressif/esp32/libraries/SD/src/SD.h:17:0,
                 from /Users/romain/Documents/Arduino/libraries/Adafruit_VS1053/Adafruit_VS1053.h:34,
                 from /Users/romain/Documents/Arduino/libraries/Adafruit_VS1053/Adafruit_VS1053.cpp:15:
/Users/romain/Documents/Arduino/hardware/espressif/esp32/libraries/FS/src/FS.h:60:12: note:   initializing argument 1 of 'size_t fs::File::read(uint8_t*, size_t)'
     size_t read(uint8_t* buf, size_t size);
            ^`

Any issue ? 
Thanks, 

Romain

--- a/Adafruit_VS1053.cpp
+++ b/Adafruit_VS1053.cpp
@@ -187,11 +187,11 @@ unsigned long Adafruit_VS1053_FilePlayer::mp3_ID3Jumper(File mp3) {
       	tag[3] = '\0';
         if (!strcmp(tag, "ID3")) {
           if (mp3.seek(6)) {
-          	start = 0ul ;
+            start = 0ul ;
             for (byte i = 0 ; i < 4 ; i++) {
               start <<= 7 ;
               start |= (0x7F & mp3.read()) ;
-  			}
+  	        }
           } else {
             //Serial.println("Second seek failed?");
           }

--- a/Adafruit_VS1053.cpp
+++ b/Adafruit_VS1053.cpp
@@ -170,28 +170,28 @@ boolean Adafruit_VS1053_FilePlayer::stopped(void) {
 
 // Just checks to see if the name ends in ".mp3"
 boolean Adafruit_VS1053_FilePlayer::isMP3File(const char* fileName) {
-    return (strlen(fileName) > 4) && !strcasecmp(fileName + strlen(fileName) - 4, ".mp3");
+  return (strlen(fileName) > 4) && !strcasecmp(fileName + strlen(fileName) - 4, ".mp3");
 }
 
 unsigned long Adafruit_VS1053_FilePlayer::mp3_ID3Jumper(File mp3) {
 
-  char 					tag[4];
-  uint32_t 			start;
-	unsigned long current;
+  char tag[4];
+  uint32_t start;
+  unsigned long current;
 	
   start = 0;
   if (mp3) {
   	current = mp3.position();
     if (mp3.seek(0)) {
-      if (mp3.read(tag,3)) {
+      if (mp3.read((uint8_t*) tag,3)) {
       	tag[3] = '\0';
         if (!strcmp(tag, "ID3")) {
           if (mp3.seek(6)) {
           	start = 0ul ;
-  					for (byte i = 0 ; i < 4 ; i++) {
-    					start <<= 7 ;
-    					start |= (0x7F & mp3.read()) ;
-  					}
+            for (byte i = 0 ; i < 4 ; i++) {
+              start <<= 7 ;
+              start |= (0x7F & mp3.read()) ;
+  			}
           } else {
             //Serial.println("Second seek failed?");
           }

